sdk/metric/exemplar/reservoir_test.go

```diff
@@ -144,3 +146,64 @@ func ReservoirTest[N int64 | float64](f factory) func(*testing.T) {
 		})
 	}
 }
+
+func reservoirConcurrentSafeTest[N int64 | float64](f factory) func(*testing.T) {
+	return func(t *testing.T) {
+		t.Helper()
+		rp, n := f(1)
+		if n < 1 {
+			t.Skip("skipping, reservoir capacity less than 1:", n)
+		}
+		r := rp(*attribute.EmptySet())
+
+		var wg sync.WaitGroup
+
+		for i := range 5 {
```

Comment:
Maybe a little bit of comment around why 5 and 2 are good numbers?

Reply:
I reduced it to 2 and 1, and documented them.